playbooks/robusta_playbooks/k8s_resource_enrichments.py

```diff
@@ -48,6 +48,7 @@ class RelatedContainer(BaseModel):
     status: Optional[str] = None
     created: Optional[str] = None
     ports: List[Any] = []
+    status_details: Optional[str] = None
```

Comment:
Can you please change to camel case to be consistent with the other class fields?

Reply:
done